File: mifosng-android/src/main/java/com/mifos/objects/accounts/loan/LoanApproval.java
/*
 * This project is licensed under the open source MPL V2.
 * See https://github.com/openMF/android-client/blob/master/LICENSE.md
 */
package com.mifos.objects.accounts.loan;

import com.google.gson.annotations.SerializedName;

public class LoanApproval {

    @SerializedName("approvedOnDate")
    String approvedOnDate;

    @SerializedName("approvedLoanAmount")
    String approvedLoanAmount;

    @SerializedName("expectedDisbursementDate")
    String expectedDisbursementDate;

    @SerializedName("note")
    String note;

    String locale = "en";
    String dateFormat = "dd MMMM yyyy";

    public String getExpectedDisbursementDate() {
        return expectedDisbursementDate;
    }

    public void setExpectedDisbursementDate(String expectedDisbursementDate) {
        this.expectedDisbursementDate = expectedDisbursementDate;
    }

    public String getLocale() {
        return locale;
    }

    public void setLocale(String locale) {
        this.locale = locale;
    }

    public String getDateFormat() {
        return dateFormat;
    }

    public void setDateFormat(String dateFormat) {
        this.dateFormat = dateFormat;
    }

    public String getApprovedOnDate() {
        return approvedOnDate;
    }

    public void setApprovedOnDate(String approvedOnDate) {
        this.approvedOnDate = approvedOnDate;
    }

    public String getNote() {
        return note;
    }

    public void setNote(String note) {
        this.note = note;
    }

    public String getApprovedLoanAmount() {
        return approvedLoanAmount;
    }

    public void setApprovedLoanAmount(String approvedLoanAmount) {
        this.approvedLoanAmount = approvedLoanAmount;
    }
}

File: core/datastore/src/main/java/com/mifos/core/objects/accounts/loan/LoanApproval.kt
/*
 * This project is licensed under the open source MPL V2.
 * See https://github.com/openMF/android-client/blob/master/LICENSE.md
 */
package com.mifos.core.objects.accounts.loan

import android.os.Parcelable
import kotlinx.parcelize.Parcelize

@Parcelize
class LoanApproval (
    var approvedOnDate: String? = null,

    var approvedLoanAmount: String? = null,

    var expectedDisbursementDate: String? = null,

    var note: String? = null,

    var locale: String = "en",

    var dateFormat: String = "dd MMMM yyyy",
) : Parcelable